src/api/app/views/webui/package/_rpmlint_log.html.haml

```diff
@@ -1,2 +1,4 @@
+- if plot_graphic
```

Comment:
What about using the word `render_chart`? We already use the word `chart` it in the `RpmLintComponent`, and we use `render` everywhere else. Using new words to describe the same will increase the cognitive load...

Reply:
Agree. Changed. Thanks!